pyproject.toml

```diff
@@ -2,49 +2,51 @@
 requires = ["poetry-core", "poetry-dynamic-versioning"]
 build-backend = "poetry_dynamic_versioning.backend"
 
-[tool.poetry]
+[project]
 name = "procrastinate"
 version = "0.0.0"
```

Comment:
I think we can use `dynamic = ["version"]`.

Reply:
Hm, I had made that change, I wonder why it was undone...

Reply:
Ah yes, the official recommendation from `poetry check` was to use
```
[project]
dynamic = ["version"]

[tool.poetry]
version = "0.0.0"
```